substrafl_examples/get_started/plot_substrafl_sklearn_fedavg.py

```diff
@@ -0,0 +1,571 @@
+"""
+=============================================
+Substrafl Scikit-Learn FedAvg on IRIS dataset
+=============================================
+
+This example illustrate an advanced usage of Substrafl as it does not use the SubstraFL PyTorch interface, but showcase the general SubstraFL interface on which you can use any ML framework.
+using the federated averaging strategy.
+
+We will implement a custom algorithm to show how to interface SubstraFL with Scikit-Learn.
```

Comment:
```suggestion
```

Reply:
You mean Substra Fl algorithm? At this stage the user does not really know what is a SubstraFl algo and might think of writing this own neural network for instance;